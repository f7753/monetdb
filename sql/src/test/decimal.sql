create table test ( t1 dec(5,2), t2 numeric(7,3));
insert into test values (100.01, 100.021);
insert into test values (90.91, 980.21);
insert into test values (100.901, 10.021);
insert into test values (9990.91, 99980.021);
select * from test;
