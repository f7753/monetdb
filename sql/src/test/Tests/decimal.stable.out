stdout of test 'decimal` in directory 'src/test` itself:


# 16:14:44 >  
# 16:14:44 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_PREFIX__GNU_32_d__/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_PREFIX__GNU_32_d__/lib/MonetDB" --set "gdk_dbfarm=/var/tmp/_PREFIX__GNU_32_d__/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_PREFIX__GNU_32_d__/var/MonetDB/log" --set mapi_port=45373 --set sql_port=50816 --set monet_prompt= --trace "--dbname=mTests_src_test" "/var/tmp/_BUILD_SQL_GNU_32_d__/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 16:14:44 >  

# Monet Database Server V4.3.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-pc-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 16:14:45 >  
# 16:14:45 >  Mtimeout -timeout 60 MapiClient --language=sql -u monetdb -P monetdb --host=draco --port=50816  < decimal.sql
# 16:14:45 >  

# 6 # querytype
# 4 # querytype
# affected # table_name
# rows # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 1 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 2 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 3 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 4 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 5 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 6 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 7 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 8 # id
[ 1	]
# 4 # querytype
#  # table_name
# single_value # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 9 # id
[ 1	]
# 4 # querytype
# affected # table_name
# rows # name
# mediumint # type
# 0 # length
# 1 # tuplecount
# 10 # id
[ 1	]
# 7 # querytype
# 7 # querytype
# 7 # querytype
# 7 # querytype
# 7 # querytype
# 3 # querytype
# test # table_name
# t1 # name
# decimal # type
# 0 # length
# 10 # tuplecount
# 10 # id
[ 1.10	]
[ -1.10	]
[ 12.12	]
[ -12.12	]
[ 123.12	]
[ -123.12	]
[ 123.12	]
[ -123.12	]
[ 123.13	]
[ -123.13	]
# 6 # querytype
# 7 # querytype

# 16:14:45 >  
# 16:14:45 >  Done.
# 16:14:45 >  

